agent/structs/acl_oss.go

```diff
@@ -14,6 +14,7 @@ import (
 
 const (
 	EnterpriseACLPolicyGlobalManagement = ""
+	EnterpriseACLPolicyGlobalReadOnly   = ""
```

Comment:
Hey @jjacobson93 ! Working on the enterprise PR to implement the read-only policy, and to do something similar as [here](https://github.com/hashicorp/consul/pull/18319/files#diff-ce26d2737c643d28b4b8486ab1b2ed56b339183d5d57de6c8e72537864ebf5ebR89-R90), these two constants will also need to be variables instead e.g. 
```
var (
	EnterpriseACLPolicyGlobalManagement = ""
	EnterpriseACLPolicyGlobalReadOnly   = ""
)
```

Reply:
I _think_ cause these are just empty strings for OSS, its fine, but yeah the enterprise PR will have them as variables.

Reply:
Ah, yes! You're right. I just tested it out and an enterprise-side var works. Thank you!